jobs/JREGIONAL_RUN_ANAL

```diff
@@ -60,7 +60,12 @@ the specified cycle.
 #
 #-----------------------------------------------------------------------
 #
-analworkdir=${CYCLE_DIR}/anal_gsi
+CYCLE_TYPE=${CYCLE_TYPE:-0}
```

Comment:
I just realized that CYCLE_TYPE isn't documented anywhere. Since it is an input argument to the jobs scripts, I'd suggest adding that information to each of the jobs that takes it as input.

Reply:
Ming changed this in https://github.com/ShunLiu-NOAA/regional_workflow/pull/13/files